### PR TITLE
fix(#579) gulp build for Windows/DOS

### PR DESCRIPTION
**Pull request by pjsenthilkumar**
@dwaynekj @joneit

This PR:
Line formatting of css on index.js and css-template fixes on gulpfile.js to fix the issue of gulp not building on Windows/DOS.

### CODE REVIEW

**Comment by Dwaynekj** [comment]
Did you test this on Mac as well @pjsenthilkumar

**Comment by pjsenthilkumar** [comment]
Yes, just checked.

File: css/index.js [hunk 1]
```diff
@@ -1,39 +1,8 @@
...
-'	margin: 0;',
-'	padding: 0 5px;',
-'	border: 0; /*border: 1px solid #777;*/',
-'	outline: 0;',
-'}',
-'',
-''
+//on one line to fix gulp build for Windows, see gulpfile.js
+'.hypergrid-container {', ' position: relative;', ' height: 500px;', '}', '.hypergrid-container > div:first-child {', ' position: absolute;', ' left: 0;', ' top: 0;', ' right: 0;', ' bottom: 0;', '}', '.hypergrid-container > div:first-child > div.info {', ' position: absolute;', ' display: none; /* initially hidden */', ' margin-top: 150px; /* to place below headers */', ' color: #eee;', ' text-shadow: 1px 1px #ccc;', ' font-size: 36pt;', ' font-weight: bold;', ' text-align: center;', ' top: 0; right: 0; bottom: 0; left: 0;', '}', '.hypergrid-textfield {', ' position: absolute;', ' font-size: 12px;', ' color: black;', ' background-color: ivory;', ' box-sizing: border-box;', ' margin: 0;', ' padding: 0 5px;', ' border: 0; /*border: 1px solid #777;*/', ' outline: 0;', '}', '', ''
+
```
Comment (Dwaynekj) [inline css/index.js]: get rid of these new lines

File: gulpfile.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ gulp.task('build', function(callback) {
     runSequence(
         'lint',
         'images',
-        'css-templates',
+        //'css-templates', //commented out to fix gulp build on Windows
```
Comment (Dwaynekj) [inline gulpfile.js]: can we leave this turned on. Otherwise it will be a breaking change

**Comment by joneit** [comment]
fyi gulp is working for me on Windows... so not sure this is needed!
- Windows 7
- Node 6.9.1
- npm 3.10.8
- gulp 3.9.1

**Comment by Dwaynekj** [comment]
Ah good to know!

**Pull request closed without merging**